src/common/enums/ui/button-mode/button-mode.ts

```diff
@@ -0,0 +1,7 @@
+enum ButtonMode {
```

Comment:
We don't need to put these files in separate folders. Let's keep other enums file structure:
 ui 
 --button-mode.ts
 --button-type.ts